client/lib/js/react/components/asset_detail.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+import { Row, Glyphicon } from 'react-bootstrap/lib';
+import classnames from 'classnames';
+
+
+const AssetDetail = React.createClass({
+    propTypes: {
+        asset: React.PropTypes.object,
+        assetContent: React.PropTypes.string,
+        children: React.PropTypes.oneOfType([
+            React.PropTypes.arrayOf(React.PropTypes.node),
+            React.PropTypes.node
+        ]),
+        className: React.PropTypes.string
+    },
+
+    getDefaultProps() {
+        return {
+            className: 'pull-right'
+        };
+    },
+
+    getAssetContent() {
+        const { asset, assetContent } = this.props;
+        if (!assetContent) {
+            return asset.transaction.data ? asset.transaction.data.payload.content : '-';
+        } else {
+            return assetContent;
+        }
+    },
+
+    render() {
+        const {
+            asset,
+            children,
+            className
+        } = this.props;
+
+        const assetContent = this.getAssetContent();
+        const validGlyph = asset.hasOwnProperty('assignee') ? <Glyphicon glyph="cog" />
```

Comment:
Then you should put a comment here. What does assignee mean in this case and why is it only there if it comes from either bigchain or the backlog?